### PR TITLE
Enhance out of stock message

### PR DESCRIPTION
**Pull request by JimmyHoenderdaal**
same as https://github.com/rapidez/checkout-theme/pull/143
but for v2

### CODE REVIEW

**Review by Roene-JustBetter — APPROVED** [review]

File: resources/views/cart/partials/product/description.blade.php [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+            <a :href="item.product.url_key + item.product.url_suffix | url" dusk="cart-item-name">@{{ item.product.name }}</a>
             <div v-for="option in item.configurable_options">
                 @{{ option.option_label }}: @{{ option.value_label }}
             </div>
             <div v-for="option in item.customizable_options">
                 @{{ option.label }}: @{{ option.values[0].label || option.values[0].value }}
             </div>
-            <div class="text-red-600" v-if="!canOrderCartItem(item)">
-                @lang('This product it out of stock, remove it to continue your order.')
+            <div v-if="!canOrderCartItem(item)" class="text-ct-error text-balance bg-ct-error/5 my-2 max-w-sm p-2">
```
Comment (BobWez98) [inline resources/views/cart/partials/product/description.blade.php:12]: Please replace the v-if with this check; https://github.com/rapidez/core/pull/627/files#diff-9da9c7eb5d8ffc35371000ac0ceb2c0f5b43725a9dfc4312be677e84a22cf3dbR18

File: resources/views/cart/partials/product/image.blade.php [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <td class="h-24 w-32 !pl-0 max-md:flex max-md:w-1/2">
-    <img v-if="item.product.image" class="object-contain" :alt="item.product.name" :src="'/storage/{{ config('rapidez.store') }}/resizes/200/magento' + item.product.image.url.replace(config.media_url, '') + '.webp'">
+    <img
+        v-if="item.product.image"
+        v-bind:class="{'opacity-20' : !canOrderCartItem(item)}"
```
Comment (BobWez98) [inline resources/views/cart/partials/product/image.blade.php:4]: Replace with 
```suggestion
        v-bind:class="{'opacity-20' : !item.is_available}"
```

**Review by BobWez98 — CHANGES_REQUESTED** [review]

**Review by jordythevulder — DISMISSED** [review]

**Review by Roene-JustBetter — DISMISSED** [review]

**Comment by royduin** [comment]
Conflicts 😇

**Review by Roene-JustBetter — APPROVED** [review]

**Review by BobWez98 — APPROVED** [review]

File: resources/views/cart/partials/product/description.blade.php [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
-            <div dusk="cart-item-name">@{{ item.product.name }}</div>
+        <div>
+            <a :href="item.product.url_key + item.product.url_suffix | url" dusk="cart-item-name">@{{ item.product.name }}</a>
             <div v-for="option in item.configurable_options">
                 @{{ option.option_label }}: @{{ option.value_label }}
             </div>
             <div v-for="option in item.customizable_options">
                 @{{ option.label }}: @{{ option.values[0].label || option.values[0].value }}
             </div>
+            <div v-if="!item.is_available" class="text-ct-error text-balance bg-ct-error/5 my-2 max-w-sm p-2">
```
Comment (royduin) [inline resources/views/cart/partials/product/description.blade.php:12]: balance?

File: resources/views/cart/partials/product/price.blade.php [hunk 4]
```diff
@@ -7,7 +7,9 @@
     </div>
 </td>
 <td class="flex items-center font-medium max-md:w-1/3 md:table-cell *:mx-auto">
-    <x-rapidez-ct::input.quantity/>
+    <template v-if="!item.is_available">
+        <x-rapidez-ct::input.quantity/>
+    </template>
```
Comment (royduin) [inline resources/views/cart/partials/product/price.blade.php:12]: Do we want this?

**Pull request closed without merging**